webextensions/api/tabs.json

```diff
@@ -720,7 +742,7 @@
                 "notes": [
                   "The default file format is 'jpeg'."
                 ],
-                "version_added": true
+                "version_added": false
```

Comment:
Is this change for Chrome intentional? Is it the same for Opera?

Reply:
Doh!